### PR TITLE
Support composable roles split into different VMSets

### PR DESCRIPTION
**Pull request by stuggi**
Depends on: https://github.com/openstack-k8s-operators/osp-director-operator/pull/162

### CODE REVIEW

File: ansible/templates/osp/ctlplane/osp-director_openstackcontrolplane.yaml.j2 [hunk 1]
```diff
@@ -28,8 +20,20 @@ spec:
                 enabled: false
             port:
             - name: {{ osp_interface }}
-          description: Linux bridge with {{ osp_interface }} as a port
-          name: br-osp
+          description: Linux bridge with enp6s0 as a port
```
Comment (stuggi) [inline ansible/templates/osp/ctlplane/osp-director_openstackcontrolplane.yaml.j2:23]: need to fix that

File: ansible/templates/osp/ctlplane/osp-director_openstackcontrolplane.yaml.j2 [hunk 2]
```diff
@@ -32,4 +24,16 @@ spec:
...
+    controller:
+      roleName: Controller
+      roleCount: {{ osp_controller_count }}
+      networks:
+        - ctlplane
+      cores: {{ osp_controller_cores }}
+      memory: {{ osp_controller_memory }}
+      diskSize: {{ osp_controller_disk_size }}
+      # FIXME: remove this once we make baseImageURL optional
+      baseImageURL: {{ osp_controller_base_image_url }}
```
Comment (abays) [inline ansible/templates/osp/ctlplane/osp-director_openstackcontrolplane.yaml.j2:37]: I think we have this functionality in the operator now.  We can leave this as-is for now, just pointing it out.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**